addons/mail/static/src/models/attachment_viewer.js

```diff
@@ -213,6 +232,16 @@ registerModel({
             }
             return style;
         },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsMultiAttachment() {
+            if (this.attachments) {
+                return this.attachments.length > 1;
+            }
+            return false;
+         }
```

Comment:
This field is kind of useless, why not check attachments length directly like before?

Reply:
Answered on other comment :)